libs/@local/hql/diagnostics/src/severity.rs

```diff
@@ -0,0 +1,77 @@
+use core::fmt::Display;
+
+use anstyle::{Ansi256Color, AnsiColor, Color};
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "serde", cfg_eval, serde_with::serde_as)]
```

Comment:
[a84f753](https://github.com/hashintel/hash/pull/4779/commits/a84f7538781f7fe8e752a4cb637f3b0893a5360c)

Reply:
If I understood it correctly, `cfg_attr` does not evaluate the proc-macro, instead using it as an attribute (e.g. `serde_with::serde_as` becomes an attribute, not a proc-macro), `cfg_eval` instead evaluates the attribute, instead of just setting it on the body to be user by other macros.

Reply:
Ah, that makes sense. That `cfg_attr` wouldn't evaluate the proc macro makes sense. Thank you!